ambient.d.ts

```diff
@@ -319,6 +344,56 @@ declare module '@mojaloop/central-services-error-handling' {
   export function ReformatFSPIOPError(error: any, apiErrorCode?: any, replyTo?: any, extensions?: any): FSPIOPError
   export function CreateFSPIOPError(apiErrorCode?: any, message?: any, cause?: any, replyTo?: any, extensions?: any, useDescriptionAsMessage?: boolean): FSPIOPError
 }
+declare module '@mojaloop/central-services-stream' {
+  import { EventEmitter } from 'events';
+  import { EventActionEnum, EventTypeEnum } from '@mojaloop/central-services-shared';
+  export interface KafkaConsumerConfig {
+    eventType: EventTypeEnum,
+    eventAction: EventActionEnum,
+    options: {
+      mode: number,
+      batchSize: number,
+      pollFrequency: number,
+      recursiveTimeout: number,
+      messageCharset: string,
+      messageAsJSON: boolean,
+      sync: boolean
+      consumeTimeout: number
+    },
+    rdkafkaConf: {
+      'client.id': string,
+      'group.id': string
+      'metadata.broker.list': string,
+      'socket.keepalive.enable': boolean
+    },
+    topicConf: {
+      'auto.offset.reset': string
+    }
+  }
+
+  // TODO: figure out a better example of a message - we may need to just do some stringifying
+  type Message = any;
```

Comment:
instead of using Message = any, let use generic function `consume` where Message is the parameter's type